_studio/shared/umc/codec/h265_dec/src/umc_h265_task_supplier.cpp

```diff
@@ -1638,6 +1650,11 @@ UMC::Status TaskSupplier_H265::ProcessNalUnit(UMC::MediaDataEx *nalUnit)
     UMC::MediaDataEx::_MediaDataEx* pMediaDataEx = nalUnit->GetExData();
     NalUnitType unitType = (NalUnitType)pMediaDataEx->values[pMediaDataEx->index];
 
+#if (MFX_VERSION >= 1025)
```

Comment:
as I understand, we need this code only if we get to line 1682. So let's put it there

Reply:
Will submit a new patch on this.